pimd/pim_iface.c

```diff
@@ -1157,17 +1157,17 @@ long pim_if_t_suppressed_msec(struct interface *ifp)
 	return t_suppressed_msec;
 }
 
-static void igmp_join_free(struct igmp_join *ij)
+static void igmp_join_free(struct gm_join *ij)
```

Comment:
When we move the datastruct name from igmp_join to gm_join, shall we change all the function names as well accordingly to accommodate both IGMP & MLD? It can be `gm_join_free` instead of `igmp_join_free`. We can update in other places as well.

Reply:
Thanks Rathna for reviewing it. We have not changed the function names yet because it is still doing igmp work. While we will implement MLD, we will change the function names if required. Few things are different between igmp and mld. We are planning to have separate file for MLD and IGMP. So whatever will be common will be renamed in terms of function when we raise the PR for MLD.

Reply:
@rathnasabapathyv I don't think it's necessary to rename the functions at the same time; it's more important to keep moving here.